scripts/sst_prep_obs.sh

```diff
@@ -2,60 +2,41 @@
 
 while getopts "i:d:" opt; do
    case $opt in
-      i) sst_source=("$OPTARG");;
-      d) filename=("$OPTARG");;
+      i) SSTsource=("$OPTARG");;
+      d) sat=("$OPTARG");;
    esac
 done
 shift $((OPTIND -1))
 
-echo SST SOURCE = ${sst_source}
-
-
+echo DCOM_ROOT=$DCOM_ROOT
 cd $DCOM_ROOT
 
-ObsRunDir=$RUNCDATE/Data/${CDATE}    #TODO: Should not be needed here ...
-OUTFILE=ioda.${sst_source}.${DA_SLOT_LEN}h.nc   #Filename of the processed obs
-PREPROCobs=${IODA_ROOT}/${CDATE}/${OUTFILE}     #FullPath/Filename of preprocessed obs
-PROCobs=${ObsRunDir}/${OUTFILE}                 #FullPath/Filename of observations to be ingested
-
-
-echo "obsrundir: "${ObsRunDir}
-echo "obsrundir: "$RUNCDATE/Data/${CDATE}
-
-#Check if the observations have been preprocessed.
```

Comment:
This cannot be deleted, it is to check for the prestaged obs,

Reply:
Will restore those line but I removed them since they were not part of the script I had and the one I tested.